### ISSUE TITLE
Python Environments Window WCAG 4.1.2: (devenv/list item 'Packages (Conda)') The Name property of sibling list items should be unique.

### ISSUE THREAD

**Issue by bschnurr** [microsoft/PTVS#5865]
The following accessibility issue needs investigation.
![image](https://user-images.githubusercontent.com/1946977/68501825-ea7e4780-0213-11ea-82fc-9dafa4024a03.png)

**App:** devenv

**Element path:** list item 'Packages (Conda)'

**Issue Details:** The Name property of sibling list items should be unique. [WCAG 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

**How To Fix:** Provide unique names for sibling list items.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**Comment by judej** [comment]
Not repro
